scripts/stop-agents.sh

```
@@ -0,0 +1,188 @@
+            
+        elif [[ "$arg" == "--help" || "$arg" == "-h" || "$arg" == "help" ]]; then
+            # ヘルプ表示
+            show_usage
+            exit 0
+            
+        else
+            # 特定タイプの全エージェントを停止
+            local agent_type=$arg
+            
+            # agent_typeの検証
+            if [[ ! " ${AGENT_TYPES[@]} " =~ " $agent_type " ]]; then
```

Shellcheck (0.10.0)

[error] Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).

(SC2199)